docs/modules/reference/pages/service-assurance/monitors/WmiMonitor.adoc

```diff
@@ -0,0 +1,132 @@
+
+= WmiMonitor
+
+The WMI monitor tests the availability of WMI services on remote interfaces.
+
+Note that WSMAN does everything that WMI does and is a more modern protocol for Windows monitoring.
+
+//verify
+This monitor implements <<reference:service-assurance/introduction.adoc#ref-service-assurance-monitors-placeholder-substitution-parameters, placeholder substitution in parameter values>>.
+
+== Monitor facts
+
+[cols="1,7"]
+|===
+| Class Name
+| `org.opennms.netmgt.poller.monitors.WmiMonitor`
+|===
+
+== Configuration and use
+
+.Monitor-specific parameters for the WmiMonitor
+[options="header"]
+[cols="1,3,2"]
+|===
+| Parameter
+| Description
+| Default
+
+3+| *Required*
+
+| username
+| The username to use when authenticating against the WMI host.
+| none
+
+| password
+| The password to use when authenticating against the WMI host.
+| none
+
+| domain
+| Windows domain where the user is located.
+You do not have to use the domain parameter if you use local user accounts.
+| none
```

Comment:
These are only required if the user has not defined credentials in `$\{OPENNMS_HOME}/etc/wmi-config.xml`
Credentials given here will override any defined in the wmi-config file.  It's probably a better practice to put credentials there than in the pollerd/collectd config.